idd/Energy+.idd.in

```diff
@@ -63383,12 +63383,15 @@ Boiler:Steam,
        \key OtherFuel1
        \key OtherFuel2
   N1 , \field Maximum Operating Pressure
-       \units Kpa
+       \required-field
+       \units Pa
   N2 , \field Theoretical Efficiency
+       \required-field
        \type real
        \minimum 0.0
        \maximum 1.0
   N3 , \field Design Outlet Steam Temperature
+       \required-field
        \units C
```

Comment:
Steam plant sizing appears to be hardcoded to use 100C. so that would be the default here I think.